pytest-selenium/README.md

```diff
@@ -0,0 +1,133 @@
+# Selenium tests for rex-web
+
+## Getting Started
+
+### Training
+
+To learn more about Pytest and Selenium please utilize the [Introduction to Pytest and Selenium][selenium-training] training wiki article.
+
+### Clone the repository
+
+If you have cloned this project already then you can skip this, otherwise you'll
+need to clone this repo using Git. If you do not know how to clone a GitHub
+repository, check out this [help page][git-clone] from GitHub.
+
+## How to prepare the project locally
+
+### Install dependencies
+
+#### Create a virtualenv
+
+    $ make venv
+
+#### Activate the virtualenv
+
+    $ source .venv/bin/activate
+
+## Run the tests using Pytest
+
+To run a specific test, pass in a value for `-k`:
+
+```bash
+$ pytest -k test_my_feature --driver Chrome pytest-selenium/tests
+```
+
+### Additional Pytest Options
+
+The pytest plugin that we use for running tests has a number of advanced
+command line options available. To see the options available, run
+`pytest --help`. The full documentation for the plugin can be found
+[here][pytest-selenium].
+
+## Uploading results to TestRail
+
+The TestRail integration is currently intended to be used during a local test run of the rex-web pytest suite when the uploading of results to TestRail is desired.
+
+Create a testrail.cfg file:
+
+    $ touch testrail.cfg
+
+Add these configs with the appropriate values:
+
+    [API]
```

Comment:
Most of the time i believe this will be ran by automation. These instructions are here in the event if someone wants to do that locally which may not be that often.